### ISSUE TITLE
Tests fail on armhf

### ISSUE THREAD

**Issue by aviau** [vmihailenco/msgpack#51]
Hello!

The test fail on armhf:
```
go test -v gopkg.in/vmihailenco/msgpack.v2 gopkg.in/vmihailenco/msgpack.v2/codes
# gopkg.in/vmihailenco/msgpack.v2_test
src/gopkg.in/vmihailenco/msgpack.v2/msgpack_test.go:66: constant 4294967296 overflows uint
src/gopkg.in/vmihailenco/msgpack.v2/msgpack_test.go:67: constant 18446744073709551613 overflows uint
src/gopkg.in/vmihailenco/msgpack.v2/msgpack_test.go:68: constant 18446744073709551614 overflows uint
src/gopkg.in/vmihailenco/msgpack.v2/msgpack_test.go:69: constant 18446744073709551615 overflows uint
src/gopkg.in/vmihailenco/msgpack.v2/msgpack_test.go:103: constant -9223372036854775808 overflows int
```

This was found by the [Debian CI](https://reproducible.debian.net/rb-pkg/unstable/armhf/golang-gopkg-vmihailenco-msgpack.v2.html)

**Comment by vmihailenco** [comment]
Most likely int/uint has 4 bytes size on armhf, but tests expect 8 bytes ints. I will update tests.

**Comment by vmihailenco** [comment]
Should be fixed in v2.4.4

**Comment by aviau** [comment]
thanks! I am uploading to Debian right away
